code/game/objects/items/tools/wrench.dm

```diff
@@ -53,22 +59,12 @@
 
 	// Let the sound effect finish playing
 	sleep(20)
-
 	if(!user)
 		return
-
 	for(var/obj/item/W in user)
 		user.dropItemToGround(W)
-
-	var/obj/item/wrench/medical/W = new /obj/item/wrench/medical(loc)
-	W.add_fingerprint(user)

```

Comment:
forensics nerf smh

Reply:
arent items logged by default? there is no need for double logging

Reply:
ur right